util/perl/TLSProxy/Proxy.pm

```diff
@@ -568,17 +614,11 @@ sub message_list
 sub serverpid
 {
     my $self = shift;
-    if (@_) {
-        $self->{serverpid} = shift;
-    }
     return $self->{serverpid};
 }
 sub clientpid
 {
     my $self = shift;
-    if (@_) {
-        $self->{clientpid} = shift;
-    }
```

Comment:
If you're making `serverpid` and `clientpid` to be accessors only, you should move them up with the rest of the accessors (same as `server_addr` and `server_port`)

Reply:
??? I did move them.

Reply:
Ah! I see! Sorry I didn't get it :-) Fixed.